data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+ACME_BASE=/var/lib/acme
+mkdir -p ${ACME_BASE}/acme/private
+
+restart_containers(){
+	for container in $*; do
+		curl -X POST \
+			--unix-socket /var/run/docker.sock \
+			"http:/containers/${container}/restart"
+	done
+}
+
+while true; do
+
+	acme-client \
+		-v -b -N -n \
+		-f ${ACME_BASE}/acme/private/account.key \
+		-k ${ACME_BASE}/acme/private/privkey.pem \
+		-c ${ACME_BASE}/acme \
+		${MAILCOW_HOSTNAME} autoconfig.${MAILCOW_HOSTNAME#*.} autoconfig.${MAILCOW_HOSTNAME#*.} ${ADDITIONAL_SAN}
```

Comment:
Yes, but you have listed `autoconfig` twice.

Also autodiscover can be done via an SRV record (what I want to test and would prefer - since it would work better for multiple domains)

Have to test this for autoconfig, but I bet there is SRV support as well...